x-pack/plugins/remote_clusters/server/config.ts

```diff
@@ -18,6 +18,7 @@ export const configSchema = schema.object({
 export type ConfigType = TypeOf<typeof configSchema>;
 
 export const config: PluginConfigDescriptor<ConfigType> = {
+  deprecations: ({ deprecate }) => [deprecate('enabled', '8.0.0')],
```

Comment:
License Management's UI can be disabled via `remoteClusters.ui.enabled: false`. Is there a way to guide users to use this setting instead?